docs/source/advancedcommands.rst

```diff
@@ -1,6 +1,117 @@
 Advanced Commands
 *****************
 
+Derived Variables
+=================
+
+Calculated Constants
+--------------------
+
+Some constants may be derived from the values of other (non-calculated) constants.
+For example, air density, viscosity, and temperature are all calculated from altitude.
+These calculated constants can be represented by a variable whose value is a function
+of the constants dictionary.
+
+.. code-block:: python
+
+    # code from t_GPSubs.test_calcconst in tests/t_sub.py
+    x = Variable("x", "hours")
+    t_day = Variable("t_{day}", 12, "hours")
+    t_night = Variable("t_{night}", lambda c: 24 - c[t_day.key], "hours")
```

Comment:
I could go either way on this, but would it be clearer to use `c[t_day]` instead of `c[t_day.key]`? At the very least, am I correct that that is supported?